contexts/did-resolution-v1.json

```diff
@@ -1,22 +1,44 @@
 {
-  "@context": [
-    {
-      "@version": 1.1
+  "@context": {
+    "@version": 1.1,
+    "id": "@id",
+    "type": "@type",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "didres": "https://w3id.org/did-resolution#",
+
+    "didDocument": "didres:didDocument",
```

Comment:
Would it make sense to make the IRI defined here set to the URL to the DID Document context?

Reply:
Hmm interesting..! I guess this would mean that the `didDocument` term here in the resolution result would then always be linked to a certain version of the DID document context, but I think that might be a good thing.

Reply:
Ohh good point about linking to specific versions. In some cases I can see it as a good thing and in others I could see it as a pain. It really depends on how we version. E.g. a minor version update should keep the same IRI, but for a v2 do we add an additional property with a different IRI (e.g. "didDocumentV2": "https://w3.org/ns/did/v2")